### DISCUSSION TITLE
Trying to use the plugin

### DISCUSSION THREAD

**Discussion by moltenform** [SebastianMC/obsidian-custom-sort#154] [Q&A] [answered]
Hi,

Thank you for adding this feature, which I've wanted for a while!

I read the documentation, but I can't seem to get the plugin to work. I'm on the latest Obsidian, 1.6.7 and turned off all other plugins. When I right-click a folder, go to Custom Sorting, and click on Bookmark for Sorting, nothing happens. I was expecting to be taken to a Bookmarks page where I could specify the order. Is there another way I'm supposed to see the bookmarks?

**Comment by moltenform** [comment]
I confirmed that the Obsidian bookmarks feature is on, I've only changed a couple settings from a fresh Obsidian install.

**Reply by moltenform** [reply, marked as answer]
Never mind, I figured it out. I was missing the step to enter the View Bookmarks page. Also, it looks like viewing bookmarks shows up in the left pane now so it didn't match the videos I was looking at here.

I can now get things to sort by

1) Turning on the plugin
1) Make the notes/folders
1) Click the toggle custom sorting button to turn it on.
1) Right click-Bookmark for sorting
1) View the bookmarks by clicking bookmarks in the upper left
1) Drag drop to rearrange as needed
1) Click the 'toggle custom sorting' button twice as a way to apply the sorting changes
1) The custom order shows up!

**Comment by SebastianMC** [comment]
Hi @moltenform 

thanks for sharing!

BTW in Obsidian you can freely rearrange the UI by drap & drop of the icons. To move the Bookmarks view to the right panel or to the bottom of left / right panel, just drag & drop its toolbar icon. You can also open Bookmarks view in the main content area in a tab, if that is convenient for you. Try it. This is how I moved the Bookmarks view in the demo video.
